### DISCUSSION TITLE
run curl "Update a Library Item's Media" -> Bad Request

### DISCUSSION THREAD

**Discussion by Gru0810** [advplyr/audiobookshelf#1375] [Q&A] [answered]
Hi, I try to update a Book "li_xxx" and set ASIN.
My curl-cmd is: ( I copied it from https://api.audiobookshelf.org/#update-a-library-item-39-s-media)
<Root.Token> will be replaced by the effectiv token Value of the ROOT-User !
curl -X PATCH "http://x.x.x.x:13378/api/items/li_xxx/media" -H "Authorization: Bearer <Root.Token>" -H "Content-Type: application/json" -d '{"metadata": {"asin": "B00GTQBKRG"}}'
When I run it I get :
![image](https://user-images.githubusercontent.com/72630160/211173822-8240e901-0e1d-4905-8216-c50f88669cf8.png)
what am i doing wrong?

**Comment by advplyr** [comment]
Have you checked the logs coming from docker or the system service if you are using the PPA?

**Comment by Gru0810** [comment, marked as answer]
I was able to use within VBA (Visual Basic for Application) CreateObject("MSXML2.XMLHTTP") and the proccess updated the ASIS properly
